Intersect.Client/Entities/Player.cs

```diff
@@ -130,6 +130,19 @@ public bool IsInMyParty(Entity entity)
             return false;
         }
 
+        public bool IsInMyParty(Guid id)
+        {
+            foreach (var member in Party)
+            {
+                if (member.Id == Id)
```

Comment:
You have `Id` (capital "i", refers to `this.Id` instead of the parameter `id`).

Reply:
No longer relevant as this was removed.
But yes, that was a typo. :)